helix-query.yaml

```diff
@@ -1,2 +1,52 @@
 version: 1
-# See https://main--helix-home--adobe.hlx.page/docs/setup-indexing.html for more information.
+
+indices:
+  blog:
+    source: html
+    fetch: https://{repo}-{owner}.hlx.page/{path}
+    include:
+      - '/blog/*.(docx|md)'
```

Comment:
So, just files in `/blog` but not underneath, e.g. `/blog/trends/test.docx`?

Reply:
ah, thanks... yes, that was not intended. better now?

Reply:
Sorry, no, I had to commit the required change:

`/**/` goes deep